### DISCUSSION TITLE
Change Background Dim in the editor to a slider or add a 100% option

### DISCUSSION THREAD

**Discussion by MattRBX** [ppy/osu#30421] [Ideas] [answered]
![image](https://github.com/user-attachments/assets/d3b78f8c-30e8-4f1a-8e10-259b744d621c)

Having background dim at all is really nice, but I want to completely dim the background just like most do during gameplay.

**Comment by peppy** [comment]
This implies that you're choosing a beatmap background that you wouldn't be happy to play with? The idea of not allowing 100% dim is that, as the mapper, you are aware of what you are exposing players to when they play the beatmap with default settings.

**Reply by Lyti4n** [reply]
Just because you can map with visual clutter doesn't mean its unreasonable to not want to have to. There is literally no background choice possible that does not make it harder to see what you are doing. Yes obviously you can just deal with it , in the same way you could get used to mapping with like a podcast or something playing making the song harder to hear that doesn't mean you should have to do that. If people mapping without the background is such a problem then you should have a big problem with not setting the background until the map is done, or changing the background of a map after its already been mapped something which happens all the time including sometimes after a map is ranked.

**Reply by peppy** [reply]
Sure let's just allow 100%. It's not like mappers care about players that aren't at 100% dim anyway. Game is not game, it's click circle serious bsns simulator.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/36847
